bdk-ffi/src/bdk.udl

```diff
@@ -82,6 +82,16 @@ enum BdkError {
   "Psbt",
 };
 
+[Error]
+interface CalculateFeeError {
+  MissingTxOut(sequence<OutPoint> out_points);
+  NegativeFee(i64 fee);
```

Comment:
This is the first time we use this pattern of defining errors as interfaces in the UDL. Can you confirm that the errors print out what we expect them to? Just making sure it all works well.

Reply:
I just added two tests in the `error.rs` file to address this. Let me know if:
- those two tests make sense to you
- that is the best place to test these two errors

... or any other thoughts you had on it too. Thanks, great suggestion!